boot.sh

```diff
@@ -19,6 +19,26 @@ set -e
 
 : ${KUBE_METADATA:=} # Without the outermost braces {}.
 
+# kubelet configs
+: ${KUBE_CLUSTER_DNS:="10.96.0.10"}
+
+read -r -d '' KUBELET_CONFIG << EOF ||:
+{
+  "kind": "KubeletConfiguration",
+  "apiVersion": "kubelet.config.k8s.io/v1beta1",
+  "staticPodPath": "/etc/kubernetes/manifests",
+  "clusterDNS": [
+    "${KUBE_CLUSTER_DNS}"
```

Comment:
If you omit the quotes here then the user can (at the expense of a little more escaping/complexity when defining `KUBE_CLUSTER_DNS`) provide multiple DNS servers. e.g. `: ${KUBE_CLUSTER_DNS:='"10.96.0.10", "10.97.0.10"'}` might work?

Reply:
Good idea! I'll include this to the PR.